### ISSUE TITLE
Bug: Constant disconnection between RetroArch and SNI client, making it impossible to play multiworld

### ISSUE THREAD

**Issue by kimimaru4000** [ArchipelagoMW/Archipelago#2230]
### What happened?

Running the ArchipelagoSNIClient and RetroArch (v1.15.0) provides me with constant reconnection loops, as the SNI client and RetroArch can't maintain a connection for some reason. I'm running Archipelago on Arch Linux 64-bit.

Screenshot below.

![Screenshot](https://github.com/ArchipelagoMW/Archipelago/assets/1388037/bd434dc7-de50-4b4d-8421-4d5c9da0d722)

[Logs](https://github.com/ArchipelagoMW/Archipelago/files/12742228/Logs.txt)

I don't have the yaml file for both players - here's [mine](https://github.com/ArchipelagoMW/Archipelago/files/12742245/Player.yaml.txt).

I have all the ports I thought I'd needed opened (23074, 55355, 8080, etc) on my system firewall. Other applications I've used can do IPC just fine.

My friend on Windows 11 does not receive this error.

### What were the expected results?

The SNI client and RetroArch should be able to maintain a stable connection and allow me to play.

### Software

While playing

**Comment by kimimaru4000** [comment]
For those reading this in the future: it turns out the pacman package, `libretro-bsnes`, is just the bsnes core. It was not bsnes-mercury-performance which is necessary for this to work.

Instead, I followed the instructions [here](https://wiki.archlinux.org/title/RetroArch#Configuration) to enable the online updater, and through that I installed the bsnes-mercury-performance. Then everything just worked!
